Helper/Data.php

```diff
@@ -250,7 +250,7 @@ protected function buildIndexData(?StoreInterface $store = null): array
             'indexName' => $this->getBaseIndexName($storeId),
             'priceKey' => '.' . $currencyCode . '.default',
             'facets' => $this->configHelper->getFacets($storeId),
-            'currencyCode' => $this->configHelper->getCurrencyCode($storeId),
```

Comment:
@damcou I wasn't sure if there was a reason for separate codes here but I thought it best to be consistent with the `priceKey`.